arrow-core/src/main/kotlin/arrow/core/extensions/const/apply/ConstApply.kt

```diff
@@ -655,8 +654,8 @@ fun <A, B> Kind<Kind<ForConst, A>, A>.product(MA: Monoid<A>, arg1: Kind<Kind<For
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "product(MA, arg1)",
-  "arrow.core.product"
+    "Cons.mapN(MA, this, arg1).map { (ab, c) -> Tuple3(ab.a, ab.b, c) }",
+    "arrow.core.Cons"
```

Comment:
```suggestion
    "Cons.mapN(MA, this, arg1).map { ab, c -> Tuple3(ab.a, ab.b, c) }",
    "arrow.core.Cons", "arrow.core.Tuple3"
```

The deprecations for `product` aren't correct. They use the old singature of `mapN`.